### DISCUSSION TITLE
How to Set Transmission Type to 01 Cyclic Synchronous When Configuring a TPDO

### DISCUSSION THREAD

**Discussion by Gzw1125** [canopen-python/canopen#553] [Q&A]
Hello,

I would like to know how to set the transmission type to 01 when configuring a TPDO. Could you please provide guidance on how to achieve this?

Thank you!

**Comment by acolomb** [comment]
Please check out the documentation for the PDO classes here:
https://canopen.readthedocs.io/en/latest/pdo.html

It shows by example how to set the `.trans_type` attribute, in your case the value should be `1`.

You didn't describe your use case, but I am assuming that you are talking about a TPDO for a remote node (a slave to the SYNC service in that case).

If you actually want to use this library to implement a local node and send a TPDO from the computer when another device has sent a SYNC, then you'll need to wire that up yourself.  Subscribe to the SYNC object and call `.transmit()` on your local PDO representation object when you see the desired point has come.  The library does not (yet) implement a full device communication logic through the `LocalNode` class, but just provides the building blocks for your application code to manage.
